contrib/pax_storage/src/test/isolation2/isolation2_unsupported_schedule

```diff
@@ -0,0 +1,7 @@
+# FIXME: MANIFEST: can't write the same table at the same time.
+# held a big write lock to avoid multiple write concurrently.
+
+test: crash_recovery
+test: resource_queue_deadlock
+test: segwalrep/commit_blocking
+test: segwalrep/fts_unblock_primary
```

Comment:
The following tests involves parallel write on pax tables, that will hang if manifest implementation is on.
test: pax/insert_policy
test: pax/max_concurrency2
test: pax/max_concurrency
test: pax/parallel_delete
test: pax/parallel_update_readcommitted
test: pax/parallel_update
test: pax/vacuum_while_insert